RELEASE-NOTES.md

```diff
@@ -1,3 +1,209 @@
+# Version 25.0
+
+## Major Highlights
+
+- **Hypercalls and New Exit Event Handlers**: Exit events now use hypercalls and
+are handled using a new `ExitHandler` class, improving flexibility and
+clarity.
+- **Improved RISC-V and Arm ISA Support**: Includes major architectural
+extensions, bug fixes, and hypervisor extension support for RISC-V.
+- **Python Utilities**: Introduction of `gem5term`, an m5term replacement in
+Python, and the `hypercall_external_signal` utility for interacting with running
+simulations via hypercalls.
```

Comment:
I believe the addition of OptionalParam and DictParam should be here as they are major additions to the user APIs